_static/css/custom.css

```diff
@@ -34,6 +34,11 @@ body {
 
 .sphinx-tabs-panel {
   padding: 1em 1em 0 1em;
+  background: inherit !important;
+}
+
+.sphinx-tabs-tab {
+  background: inherit !important;
```

Comment:
Is this the workaround for https://github.com/executablebooks/sphinx-tabs/issues/194 ?